scheduler/scheduler.go

```diff
@@ -45,6 +45,10 @@ const (
 var (
 	authProvider = flag.String("mesos_authentication_provider", sasl.ProviderName,
 		fmt.Sprintf("Authentication provider to use, default is SASL that supports mechanisms: %+v", mech.ListSupported()))
+	schedulerHost = flag.String("mesos_scheduler_ip", "0.0.0.0",
+		fmt.Sprintf("TCP4 Address for the Scheduler to bind"))
```

Comment:
@fmpwizard sorry, i'm not familiar with go vet. i usually just use https://github.com/fatih/vim-go which i think is gofmt. let me see if i can fix... can you paste the error?